### PR TITLE
SONARJAVA-2135 Log classes not found once by analysis

### PR DESCRIPTION
**Pull request by benzonico**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BytecodeCompleter.java#L46](https://github.com/SonarSource/sonar-java/blob/e1a0345c14104a5965f7eb40082623cfd85c7793/java-frontend/src/main/java/org/sonar/java/resolve/BytecodeCompleter.java#L46): Remove this unused "LOG" private field. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: java-frontend/src/main/java/org/sonar/java/model/VisitorsBridge.java [hunk 1]
```diff
@@ -179,4 +183,9 @@ public void setCurrentFile(File currentFile) {
     this.currentFile = currentFile;
   }
 
+  public void endOfAnalysis() {
+    if(!classesNotFound.isEmpty()) {
+      LOG.warn("Classes not found during the analysis : {}", classesNotFound.stream().collect(Collectors.joining(", ")));
```
Comment (m-g-sonar) [inline java-frontend/src/main/java/org/sonar/java/model/VisitorsBridge.java]: Maybe add brackets around the list of classes? 
```
"Classes not found during the analysis : [x.y.z.myClass, x.z.MyOtherClass, ...]"
```

I also wonder about the possible volume of data we may end up displaying (analysis poorly configured?). Maybe displaying only the first 50 classes, and if more display all of them in debug? Splitting hairs...

Comment (benzonico) [inline java-frontend/src/main/java/org/sonar/java/model/VisitorsBridge.java]: Actually not so much. This makes sense.

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [BytecodeCompleter.java#L46](https://github.com/SonarSource/sonar-java/blob/ffe941120a34a70cf6175c71d6f58ea0b1cfda1a/java-frontend/src/main/java/org/sonar/java/resolve/BytecodeCompleter.java#L46): Remove this unused "LOG" private field. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Pull request merged**